lib/hammer_cli_foreman/hostgroup.rb

```diff
@@ -83,6 +83,7 @@ class InfoCommand < HammerCLIForeman::InfoCommand
         field nil, _("Puppet CA Proxy"), Fields::SingleReference, :key => :puppet_ca_proxy
         field nil, _("Puppet Master Proxy"), Fields::SingleReference, :key => :puppet_proxy
         field nil, _("ComputeProfile"), Fields::SingleReference, :key => :compute_profile
```

Comment:
It's not so related to this PR, but if you're here, could you please add a space here? So it becomes `_('Compute Profile')`.

Reply:
Done! :)
Sorry for the wrong title above. I did amend the commit but forgot to modify the PR title.